lib/jazz_fingers.rb

```diff
@@ -17,18 +32,37 @@ module JazzFingers
   # libedit-based wrapper (standard on OS X unless ruby is explicitly compiled
   # otherwise).
   #
-  mattr_accessor :colored_prompt
-  self.colored_prompt = (Readline::VERSION !~ /EditLine/)
+  def self.colored_prompt
+    @colored_prompt ||= Readline::VERSION !~ /EditLine/
+  end
+
+  def self.colored_prompt=(colored_prompt)
+    @colored_prompt = colored_prompt
+  end
 
   # Separator between application name and input in the prompt.
   #
   # Default: right angle quote, or '>' when using rb-readline which doesn't
   # handle mixed encodings well.
   #
-  mattr_accessor :prompt_separator
-  self.prompt_separator = defined?(RbReadline) ? '>' : "\u00BB"
+  def self.prompt_separator
+    @prompt_separator ||= defined?(RbReadline) ? '>' : "\u00BB"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.